common/util.go

```diff
@@ -11,21 +12,28 @@ import (
 	"math/big"
 	"math/rand"
 	"mime"
+	"path/filepath"
 	"regexp"
 	"sort"
 	"strconv"
 	"strings"
 	"testing"
 	"time"
 
+	"github.com/abema/go-mp4"
+	"github.com/ebml-go/webm"
 	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/go-audio/wav"
```

Comment:
Nit: Have you considered using https://github.com/gopxl/beep? Seems to convert more formats + have more GitHub stars.